templates/repo/settings/options.tmpl

```diff
@@ -14,7 +14,7 @@
 						{{.CsrfTokenHtml}}
 						<input type="hidden" name="action" value="update">
 						<div class="required field {{if .Err_RepoName}}error{{end}}">
-							<label for="repo_name">{{.i18n.Tr "repo.repo_name"}}<span class="text red hide" id="repo-name-change-prompt"> {{.i18n.Tr "repo.settings.change_reponame_prompt"}}</span></label>
+							<label for="repo_name">{{.i18n.Tr "repo.repo_name"}}</label>
```

Comment:
The prompt warns that "This change will affect how links relate to the repository". Once we have redirects, what the prompt says will no longer be true.